app-android/src/androidTest/java/com/ustadmobile/port/android/view/ReportEditFragmentTest.kt

```diff
@@ -199,22 +208,22 @@ class ReportEditFragmentTest {
 
             report.chartType.takeIf { it != reportOnForm?.chartType }?.also {
                 setMessageIdOption(R.id.fragment_edit_report_dialog_visual_type_textinputlayout,
-                        it, fragmentScenario.letOnFragment { it.chartOptions }!!)
+                        impl.getString(ReportEditPresenter.ChartOptions.values().find { report -> report.optionVal == it }!!.messageId, context))
```

Comment:
On setMessageIdOption - can that function be adjusted so that it just takes a string or string id (e.g. R.string.bar_chart etc).

Reply:
This is actually fine and required because the fillFields method takes a reportWithFilters object.